fili-core/src/main/java/com/yahoo/bard/webservice/table/resolver/AggregatableDimensionsMatcher.java

```diff
@@ -43,7 +43,7 @@ public AggregatableDimensionsMatcher(DataApiRequest request, TemplateDruidQuery
 
     @Override
     public boolean test(PhysicalTable table) {
-        Set<String> columnNames = TableUtils.getColumnNames(request, query.getInnermostQuery(), table);
+        Set<String> columnNames = TableUtils.getColumnNames(request, query.getInnermostQuery());
```

Comment:
Fun! I know that we have the `AggregatableDimensionMatcher` in our matching chain, but none of our dimensions are non-aggregatable.